src/claim.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Deserialize, Serialize};
+use serde_cbor::Value;
+
+/// Registered CWT claims from the
+/// [CWT Claims registry](https://www.iana.org/assignments/cwt/cwt.xhtml).
+pub trait Claim: Into<Value> {
```

Comment:
I'd suggest we don't label this as "Registered CWT claims", as a consumer of this library may want to use the trait for non-registered claims.